apps/rush/src/RushVersionManager.ts

```diff
@@ -50,7 +55,14 @@ export default class RushVersionManager {
 
     if (isLegacyRushVersion) {
       return new RushWrapper(() => {
-        require(path.join(expectedRushPath, 'node_modules', '@microsoft', 'rush', 'lib', 'rush'));
+        require(path.join(
+          expectedRushPath,
+          'node_modules',
+          '@microsoft',
+          'rush',
+          'lib',
+          semver.eq(version, RUSH_START_JS_VERSION) ? 'start' : 'rush'
```

Comment:
>? [](start = 52, length = 1)

Is there a way to make the logic a little more readable.  The white board would be like:

```
if (version >= 4.0.0) {
  // The new way
        const rushCliEntrypoint: typeof rushLib = require(path.join(
          expectedRushPath,
          'node_modules',
          '@microsoft',
          'rush-lib',
          'lib',
          'index'
        ));
        rushCliEntrypoint.Rush.launch(this._currentPackageVersion, true);
} else if (version == 3.0.20) {
  // The one weird release while we were working on this
 require(path.join(
          expectedRushPath,
          'node_modules',
          '@microsoft',
          'rush',
          'lib',
          'start'
        ));
} else {
  // The old way
 require(path.join(
          expectedRushPath,
          'node_modules',
          '@microsoft',
          'rush',
          'lib',
          'rush'
        ));
}
```